file.go

```diff
@@ -145,3 +149,10 @@ func (f *file) addService(s Service) {
 }
 
 func (f *file) addMapEntry(m Message) { panic("cannot add map entry directly to file") }
+
+func (f *file) lookupComments(name string) string {
+	if c, ok := f.comments[name]; ok {
+		return c
+	}
+	return ""
```

Comment:
this can probably just be simplified to `return f.comments[name]` no?

Reply:
I think so, will do.